resources/xlf/LocProject.json

```diff
@@ -1,93 +1,93 @@
 {
-  "Projects": [
-    {
-      "LanguageSet": "AzureKatal_Languages",
-      "LocItems": [
-        {
-          "SourceFile": "resources\\xlf\\en\\admin-tool-ext-win.xlf",
-          "LclFile": "resources\\localization\\LCL\\{Lang}\\admin-tool-ext-win.xlf.lcl",
-          "CopyOption": "LangIDOnName",
-          "OutputPath": "resources\\xlf\\Localized"
-        },
+	"Projects": [
```

Comment:
It looks like you've modified the file now with the latest changes - make sure you're copying the file exactly as it was

Reply:
it doesn't let me commit when I have the file as is, I did format document. Is that ok?

Reply:
Please don't modify files (especially ones not used by our team).

You should checkout main and then copy that version of the file back into this branch.